public/components/settings/api/add-api.js

```diff
@@ -93,6 +97,20 @@ export const AddApi = withErrorBoundary (class AddApi extends Component {
           'Wazuh API not reachable, please review your configuration',
         fetchingData: false
       });
+
+      const options = {
+        context: errorContext,
+        level: UI_LOGGER_LEVELS.ERROR,
+        severity: UI_ERROR_SEVERITIES.UI,
+        store: true,
+        error: {
+          error: error,
+          message: error.message || error,
+          title: `Wazuh API not reachable, please review your configuration: ${error.message || error}`,
+        },
+      };
+
+      getErrorOrchestrator().handleError(options);
```

Comment:
the error is handled by two different logics, I think only one must be the responsible for doing it.

Reply:
In this case, the orchestrator will save the error in our `.log` and write a `loglevel.error` in the console. (because it is UI severity) like this:
![image](https://user-images.githubusercontent.com/11060558/123161408-bafe6c00-d445-11eb-9fc5-94690598df17.png)